packages/components/src/Loading/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -24,60 +106,16 @@ export const LoadingSpinner = ({
 }: LoadingSpinnerProps): JSX.Element => (
   <div
     data-automation-id="loading-spinner"
```

Comment:
You reckon we could remove this? 👀

Reply:
Likely, I did a quick scan and the [closest usages](https://github.com/search?q=org%3Acultureamp+%22loading-spinner+NOT+repo%3Acultureamp%2Fkaizen-design-system+NOT+repo%3Acultureamp%2Fkaizen-legacy&type=code) I found don't seem like they should actually be impacted